### PR TITLE
did the changes

### PR DESCRIPTION
**Pull request by varun7singh**

### CODE REVIEW

**Comment by varun7singh** [comment]
@Dipti821  can you review this please?

File: src/components/TextForm.js [hunk 1]
```diff
@@ -248,13 +264,6 @@ export default function TextForm(props) {
         >
           Clear Text
         </button>
-        <button
-          disabled={text.length === 0}
-          className="btn btn-primary mx-1 my-1"
-          onClick={handleCopy}
-        >
-          Copy Text
-        </button>
```
Comment (Dipti821) [inline src/components/TextForm.js:257]: You have deleted copy text button.

File: src/components/TextForm.js [hunk 2]
```diff
@@ -293,13 +302,6 @@ export default function TextForm(props) {
         >
           Remove Line Break
         </button>
-        <button
-          disabled={text.length === 0}
-          className="btn btn-primary mx-1 my-1"
-          onClick={downloadFile}
-        >
-          Download file
-        </button>
```
Comment (Dipti821) [inline src/components/TextForm.js:302]: You have deleted download file button

**Review by Dipti821 — COMMENTED** [review]
Fetch and pull from upstream before you push.

**Comment by Dipti821** [comment]
@varun7singh are u working on it?

**Pull request closed without merging**